Makefile

```
@@ -168,17 +185,26 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 
 .PHONY: build-installer
 build-installer: manifests generate kustomize ## Generate a consolidated YAML with CRDs and deployment.
-	mkdir -p build dist
+	mkdir -p build deploy/all-in-one
 	cd build && echo "$$BUILD_INSTALLER_OVERLAY" > kustomization.yaml
-	$(KUSTOMIZE) build build > dist/install.yaml
+	$(KUSTOMIZE) build build > deploy/all-in-one/install.yaml
+	rm -rf build
+
+.PHONY: build-custom-installer
```

Just adding another example for the case to change then name:
```suggestion
.PHONY: build-multifile-installer
```